frontend/javascripts/oxalis/view/action-bar/starting_job_modals.tsx

```diff
@@ -611,9 +613,10 @@ function JobCreditCostInformation({
                   if (jobCreditCostPerGVx != null && currentBoundingBoxVolume == null) {
                     return Promise.reject("This is a requires a selected bounding box.");
                   } else if (jobCreditCostPerGVx != null && !orgaHasEnoughCredits) {
+                    /* TODO: Uncomment after credit system testing phase.
```

Comment:
reminder for this todo (todom?)

Reply:
No this should be kept IMO to make it easier to make the credit system acutally requiring payment easier. Please see https://github.com/scalableminds/webknossos/pull/8352#issuecomment-2742866271 and the linked comment & slack thread (linked in th linked comment)

Reply:
I you want to avoid this uncommented code, I could also delete it and re-add it in the pr fully enabling the credit system

Reply:
ah, I see, I thought the review process is the testing phase, but this means something different. then, nevermind :)